ci/tasks/generate-charts/generate_charts.py

```diff
@@ -121,6 +121,7 @@ def generate_chart(prefix, chart_destination: str, file_paths: List[str], simpli
         i = i + 1
 
         fig.align_ylabels()
+        fig.set_size_inches(width, 3 * len(test_results))
```

Comment:
This is already set in line 93. Does it fix the issue to adjust the value there?